compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -34,50 +85,15 @@ static LogicalResult setRootConfig(func::FuncOp entryPointFn,
           "dim");
     }
   }
-
   // TODO (nmeshram) : This needs to be moved in a separate more generalized
   // logic. Also, need a flag to experiment between pad based and pack based
   // approach which will have different tile sizes and pass pipelines
   if (usePassPipeline == AIEPassPipeline::PadPipeline) {
-    SmallVector<int64_t> TileSizeLevel0 = {8, 8};
-    SmallVector<int64_t> TileSizeLevel1 = {4, 4};
-    SmallVector<int64_t> TileSizeLevel2 = {0, 0, 4};
-    TileSizesListType tileSizes = {TileSizeLevel0, TileSizeLevel1,
-                                   TileSizeLevel2};
-    return setOpConfigAndEntryPointFnTranslation(
-        entryPointFn, matmulOp, tileSizes,
-        IREE::Codegen::DispatchLoweringPassPipeline::None);
+    return setRootConfigForPadPipeline(entryPointFn, matmulOp);
   } else if (usePassPipeline == AIEPassPipeline::SimplePackPipeline) {
```

Comment:
Nit: This could just be an `if`. This is LLVM convention to use `if` + `return` instead of `if-else`.